### DISCUSSION TITLE
hermite: interpolation not in center interval

### DISCUSSION THREAD

**Discussion by xander9797** [gmtsar/gmtsar#927] [Q&A] [answered]
hi everyone,when i run some .csh files,this warning will appear in the software,how can i solve ti
![QQ图片20240605111507](https://github.com/gmtsar/gmtsar/assets/171759911/29e3cc86-3f16-47e3-aa35-ebbbaabd8fbb)

**Comment by Xiaohua-Eric-Xu** [comment]
Feels like an orbit issue but we need more info (e.g., a log, including what you did) to determine what really happened.

**Comment by xander9797** [comment]
It seems that the reason is that my cropping range is too small, and I need to manually change the c file in gmtsar. After making the changes, this problem will no longer occur

**Reply by Xiaohua-Eric-Xu** [reply, marked as answer]
I think I made a change years ago to deal with one burst scenario. But I don't remember with branch. If you try a newer version of GMTSAR, will the error persist?

**Reply by xander9797** [reply]
This will not be the case with the new version
